engine/api/error.go

```diff
@@ -10,8 +10,15 @@ import (
 // WriteError is a helper function to return error in a language the called understand
 func WriteError(w http.ResponseWriter, r *http.Request, err error) {
 	al := r.Header.Get("Accept-Language")
-	msg, code := sdk.ProcessError(err, al)
+	msg, errProcessed := sdk.ProcessError(err, al)
 	sdkErr := sdk.Error{Message: msg}
-	log.Warning("%-7s | %-4d | %s \t %s", r.Method, code, r.RequestURI, err)
-	WriteJSON(w, r, sdkErr, code)
+
+	// ErrAlreadyTaken is not useful to log in warning
+	if errProcessed.ID == sdk.ErrAlreadyTaken.ID {
+		log.Debug("%-7s | %-4d | %s \t %s", r.Method, errProcessed.Status, r.RequestURI, err)
+	} else {
+		log.Warning("%-7s | %-4d | %s \t %s", r.Method, errProcessed.Status, r.RequestURI, err)
```

Comment:
I find it really interesting to see that there is a lot of contention on every jobs, It means we still have things to improve.
If you are upset with the warning, the log .Info would be fine for me if we are currently using the info log-level

Reply:
yes, Info